src/test/java/bio/terra/janitor/integration/common/configuration/TestConfiguration.java

```diff
@@ -11,11 +12,30 @@
 public class TestConfiguration {
   private String trackResourceTopicId;
 
+  private String clientSaPath;
+
   public String getTrackResourceTopicId() {
     return trackResourceTopicId;
   }
 
   public void setTrackResourceTopicId(String trackResourceTopicId) {
     this.trackResourceTopicId = trackResourceTopicId;
   }
+
+  public String getClientSaPath() {
+    return clientSaPath;
+  }
+
+  public void setClientSaPath(String clientSaPath) {
+    this.clientSaPath = clientSaPath;
+  }
+
+  public ServiceAccountCredentials getClientGoogleCredentialsOrDie() {
```

Comment:
add a comment on what "client" credentials are used for and what we expect them to have credentials to do.